dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -246,6 +246,168 @@
 			status = "disabled";
 			label = "RNG";
 		};
+
+		timers1: timers@50012c00 {
```

Comment:
You're using secure address space (`_S`)
This is wrong as Zephyr application isn't secure.
You should use non secure `_NS` address ranges

Reply:
Resolved.